src/Element_TEST.cc

```diff
@@ -230,7 +230,7 @@ TEST(Element, AddAttribute)
   ASSERT_TRUE(param->IsType<float>());
   ASSERT_EQ(param->GetKey(), "attr");
   ASSERT_EQ(param->GetTypeName(), "float");
-  ASSERT_EQ(param->GetDefaultAsString(), "0");
+  ASSERT_EQ(param->GetDefaultAsString(), "0.0");
```

Comment:
In https://github.com/ignitionrobotics/sdformat/pull/689/commits/4667c4bdf0dadc8ec8d2db0952778e109ea299e4, I've done a few things,
* By default all types (except for pose), will use `ParamStreamer`, which is the old behavior, thus this single line change has been reverted
  * Older tests in `test/integration/default_elements.cc` have now been reverted back to how they were in `sdf11`, https://github.com/ignitionrobotics/sdformat/blob/sdf11/test/integration/default_elements.cc

* Aside from `pose`, a type specific parsing for `bool` has been added to, to ensure we get `true` or `false` instead of `1` or `0`.
  * This required some tests in `test/integration/pose_1_9_sdf.cc` to be modified.